worker/raft/raftleaseconsumer/handler.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
Without a new line after this, my linter complaints because it thinks it's a package comment of bad form.